src/components/SidebarWithHeader.tsx

```diff
@@ -178,7 +177,6 @@ const MobileNav = ({ onOpen, ...rest }: MobileProps) => {
             px={{ base: 4, md: 4 }}
             height='20'
             alignItems='center'
-            bg={useColorModeValue('white', 'gray.900')}

```

Comment:
![image](https://user-images.githubusercontent.com/952050/194368267-35eb4b50-c760-4e37-b835-b7783413c37b.png)

something broke here, not sure what styling/changes did it

Reply:
FYI, this is only broken on the player overview page

Reply:
fixed